app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    post = Post.find(params[:post_id])
+    @comment = Comment.new(params.require(:comment).permit(:text).merge(author_id: current_user.id, post_id: post.id))
+
+    respond_to do |format|
+      format.html do
+        if @comment.save
+          redirect_to user_post_path(current_user, post)
+        else
+          redirect_to new_user_post_comment_path(current_user)
+        end
+      end
```

Comment:
### Optional

- [ ] It would be nice if we create a params method. Here is a [link](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) to help us appreciate the point